protocol/x/vault/types/params.go

```diff
@@ -10,7 +10,7 @@ import (
 func DefaultParams() Params {
 	return Params{
 		Layers:                           2,                            // 2 layers
-		SpreadMinPpm:                     10_000,                       // 100 bps
+		SpreadMinPpm:                     3_000,                        // 30 bps
```

Comment:
nit: Is there a need to change this at all?

Reply:
wanted dev/staging to mirror spread in prod

Reply:
Can you do that via the genesis rather than via the code here?

Reply:
sure that works too